Requirements/ViewModels/RequirementBrowser/Rows/RequirementRowViewModel.cs

```diff
@@ -82,6 +91,19 @@ public class RequirementRowViewModel : CDP4CommonView.RequirementRowViewModel, I
         /// </summary>
         private bool isParametricConstraintDisplayed;
 
+        /// <summary>
+        /// Gets or sets the <see cref="CDP4RequirementsVerification.RequirementStateOfCompliance"/>
+        /// </summary>
+        public RequirementStateOfCompliance RequirementStateOfCompliance
+        {
+            get => this.requirementStateOfCompliance;
+            set
+            {
+                this.RaiseAndSetIfChanged(ref this.requirementStateOfCompliance, value);
+                this.parametricConstraints.RequirementStateOfCompliance = value;
```

Comment:
not so happy with this pattern, the reactive properties should just be that. Any logic that depends need to be implemented in a different manner. please listen to changes on this property in the constructor and have logic do work based on that.